bin/check_ciao_version

```diff
@@ -188,7 +188,7 @@ def compare_caldb():
         cinfo += f" (link to {lname})"
 
     if rval is None:
-        print(f"{ciao} is up to date.")
+        print(f"{caldb} is up to date.")
```

Comment:
I think I probably meant it to be

```
print(f"{cinfo} is up to date.")
```

but I'm not sure how bothered I am about it now.